functions/EmuScripts/emuDeckPCSX2QT.sh

```diff
@@ -15,9 +15,9 @@ PCSX2QT_install() {
 	echo "Begin PCSX2-QT Install"
 	local showProgress="$1"
 	
-	if installEmuAI "pcsx2-Qt" "https://github.com/PCSX2/pcsx2/releases/download/v1.7.4749/pcsx2-v1.7.4749-linux-appimage-x64-Qt.AppImage" "" "$showProgress"; then #pcsx2-Qt.AppImage	
-	#if installEmuAI "pcsx2-Qt" "$(getReleaseURLGH "PCSX2/pcsx2" "Qt.AppImage")" "" "$showProgress"; then #pcsx2-Qt.AppImage
-		:
+	if installEmuAI "${PCSX2QT_emuName}" "https://github.com/PCSX2/pcsx2/releases/download/v1.7.4749/pcsx2-v1.7.4749-linux-appimage-x64-Qt.AppImage" "pcsx2-Qt" "$showProgress"; then # pcsx2-Qt.AppImage - filename capitalization matters for ES-DE to find it
+	#if installEmuAI "${PCSX2QT_emuName}" "$(getReleaseURLGH "PCSX2/pcsx2" "Qt.AppImage")" "pcsx2-Qt" "$showProgress"; then #pcsx2-Qt.AppImage
+		rm -rf $HOME/.local/share/applications/pcsx2-Qt.desktop &>> /dev/null # delete old shortcut
```

Comment:
Why are we deleting the desktop shortcut in the installation for pcsx2qt?

Reply:
oh - because it's the wrong capitalization?

Reply:
Yes, keeping things tidy.